searx/static/themes/simple/src/less/preferences.less

```diff
@@ -196,6 +190,18 @@
       width: 100%;
     }
   }
+
+  #tab-content-query {
+    table td,
+    table th {
+      text-align: left;
```

Comment:
It should be `.ltr-text-align-left();` to deal with RTL languages.

Reply:
thanks / done!